resources/wikia/modules/flowTracking.js

```diff
@@ -0,0 +1,58 @@
+define('wikia.flowTracking', ['wikia.cookies', 'wikia.tracker', 'mw'], function (cookies, tracker, mw) {
+	var cookieName = 'flowTrackingLabel',
```

Comment:
So if I understand correctly, we can track only one flow at a given moment. Is that intended? Or maybe it'd be better to allow for multiple trackers, each responsible for a single flow ?